src/styles/globals.css

```diff
@@ -1,4 +1,5 @@
 @import url('./tailwind.css');
+@import './hljs-github-dark.css';
```

Comment:
I tried NPM installing the underlying package instead of copying the source code from GitHub, but it looks like Contentlayer customised GitHub's CSS a fair bit to work with [HLJS](https://highlightjs.org/)